tests/test_store/test_core.py

```diff
@@ -119,6 +120,8 @@ async def test_make_store_path_invalid() -> None:
         await make_store_path(1)  # type: ignore[arg-type]
 
 
+@pytest.mark.skipif(not importlib.util.find_spec("requests"), reason="requires requests")
+@pytest.mark.skipif(not importlib.util.find_spec("aiohttp"), reason="requires aiohttp")
```

Comment:
Might be a bit nicer to use [`@pytest.importorskip`](https://docs.pytest.org/en/stable/reference/reference.html#pytest.importorskip) here?

Reply:
Ah, good call, I missed that I could use that on the test (rather than module) level

Reply:
Done in [`b2442ba` (#2872)](https://github.com/zarr-developers/zarr-python/pull/2872/commits/b2442bab8277247ef530805bb70b0bedd9b52503)